test/cypress/integration/settings/your-psp-stripe-tasklist.cy.js

```diff
@@ -261,4 +261,24 @@ describe('Your PSP Stripe page', () => {
       cy.get('strong[id="task-sro-status"]').should('contain', 'complete')
     })
   })
+
+  describe('Government entity task', () => {
+    it('should have Government entity document hyperlinked  when all other tasks are complete and should click Government entity task and display the correct page', () => {
```

Comment:
This test partially duplicates "should have government entity document hyperlinked when all other tasks are complete".

Can we remove the other one, as this covers what is tested by the other test?

Reply:
Yes I forgot to remove that